_sass/_team.scss

```diff
@@ -37,22 +37,43 @@
 .team-member-content{
     @media (max-width: $desktop-width) {
         > h3 {
-            font-size: 1.5em;
+            font-size: 1em;
+            word-wrap: break-word;
```

Comment:
We may want to investigate applying `word-wrap: break-word` more liberally everywhere, but for now we can probably nest this under `.team-member-content` to reduce duplication!